### ISSUE TITLE
Kezan Start Zone

### ISSUE THREAD

**Issue by CrossUCI** [Arkania/ArkCORE-NG#276]
Hi people, i try it to test goblin zone, but when the presentation is finished, the npc not appears so also there is no quest, i activate the gm mode and the npc are there, and the npc is replicated 3 time, what can i do, i read the txt in google drive about the mmaps, vmaps, maps , dbc, i download all , extracted, but the error continues

**Comment by gpn39f** [comment]
it seems the phasing is wrong. ( it's ok you see npc many times.. each of them has a different phase)
the newest core and db is the snapshot from septemper 17..
are u using the prebuild exe or have u compiled NG from github?

**Comment by CrossUCI** [comment]
I used the releasex64 in the folder of september 17

**Comment by gpn39f** [comment]
and database from same folder?
i will recheck both ..

**Comment by CrossUCI** [comment]
yes i download the database too, to export the world was 13 hours row by row with heidisql, because the fast way in console using mysql.exe -u user -p ng_world <ng_world.sql with this way i have a error
ERROR 1406 (22001) at line 1812693: Data too long for column 'subname_loc8' at row 1, so i have to do it the long way

**Comment by gpn39f** [comment]
i use only the mysql console.. (or navicat)
both without errors.. both only few minutes..
the sql server u use, has to be old version 5.5.xx...this is important.!!!! 
the mysql.dll and cli has to be from this old version..

**Comment by CrossUCI** [comment]
i have navicat i have to test, i used mysql community server 5.5.7

**Comment by gpn39f** [comment]
5.5.7. is fine..

**Comment by CrossUCI** [comment]
You can rebuild the release for download ?

**Comment by CrossUCI** [comment]
yes you all right, with navicat no errors to import the ng_world, and very fast

**Comment by gpn39f** [comment]
new created goblin women mage.. 
all is fine..

**Comment by CrossUCI** [comment]
ng_world have a lot of errors in the last folder, 364M only, tell me if im wrong, the file is create 06-09-2017  but it seems incomplete

**Comment by gpn39f** [comment]
should be less then 1000 byte.. if i remember was 4 errors left..
so youre database seems to be very damaged..
look at the server, into log files of mysql..
mostly failure is "server is gone away"
how to prevent you find manual in the google/folders..

**Comment by CrossUCI** [comment]
i just say that ng_world.zip compress is 68.3Mb, and uncompress 364,2, at least for me, when i download, i want to you to tell me if i am wrong, or is my download

**Comment by gpn39f** [comment]
yes.. the uncompressed database has a size about 360 mb.. 
and after loading core, the error log is less 1 k

**Comment by CrossUCI** [comment]
thanks frank, i believe that my server is very crazy, with navicat a lot of errors and with heidisql not even a single error , i just cant believe

**Comment by gpn39f** [comment]
reading log file win's..
so.. read the mysql log files inside youre server..
to see what happens..

**Comment by CrossUCI** [comment]
I really glad to say, everything is work, all fine jejeje, thanks frank, keep in touck

**Comment by gpn39f** [comment]
please tell other what you have done.. what was wrong on youre settings..

**Comment by CrossUCI** [comment]
I just import the database with heidisql, i don not why with navicat did not work, I found a mystake at less in my server The Mission in Kezan 447, when you recolect the things inside the house , and you are going to delivery the quest the npc Claim Adjuster is not there, it most be wrong phaseid because in mode gm is there

**Comment by gpn39f** [comment]
on deliver last quest, you have to let drive Sassy Hardwrench to the ship.
you are only passenger in here's car..
at end, you listen to the last part of history..

**Comment by CrossUCI** [comment]
i am not there jet,the npc Claim Adjuster  only you can see in gm mode, the carbonite indicate that the quest 447 end in a location y go there and is not there, i active the gm and is there, so i believe that is a wrong phase id, how i can fix that, if i dont fixed i can not forward

**Comment by gpn39f** [comment]
on quest 447 you have todo 3 parts inside house..
you going outside and click on the machine 
the rob going inside house and all fire up..
you take the car to ship..

**Comment by CrossUCI** [comment]
i do everything i enter into the house , get 3 parts, go outside and clic in the machine, the robot appears and stay there , it do not going inside the , it just the quest say Speak with the Claims Adjust at KTC Headquarters on Kezan, but the npc is not there

**Comment by gpn39f** [comment]
if the robot spawns, he has to go inside..
is same script..
i look tomorrow..

**Comment by CrossUCI** [comment]
thanks, i will try it to make a video , so you can see what im try it to tell you, thanks again for everything

**Comment by CrossUCI** [comment]
hello , so you looking the script or see if i am of the problem, or is the script, greettings

**Comment by gpn39f** [comment]
ok.. problem is, the bot has 4 vehicle_accessory, but i missing the loading of them..
i have to find out what happens..
this is not done "to day".. please give me time..

**Comment by CrossUCI** [comment]
i understand , don't worry, thanks for the review, and i be waiting when is done, and also you can watch for the npc who complete the quest do not appear without gm mode, thanks

**Comment by CrossUCI** [comment]
it works thank you, but , the other quest Life Savings you mounting as a passenger in the car with Lady Sassy but the car is not going to the port, and you can not complete the quest, and also the only way to leave the car is loggin out.

**Comment by gpn39f** [comment]
i look at this :-)

**Comment by CrossUCI** [comment]
hello frank, i know that you have problems recently with your operative system, i hope everything is ok, i have to ask, do you look the quest Life Savings, thanks and greettings

**Comment by gpn39f** [comment]
yes i do.. sally is not enter the car.. she stand inside, but is not sitting down.. so she is no passenger and the scripts await sally as driver.. all data should be ok.. so i look now inside the core what is wrong. i have changed much inside the transport and vehicle system so maybe is a mistake here..

**Comment by CrossUCI** [comment]
thanks for answer, i hope you find it

**Comment by CrossUCI** [comment]
nice, im glad to read that, well frank what about the core, with the quest Saving Life

**Comment by CrossUCI** [comment]
Hello Frank i download the source code , i compile it and test, so i start again to test goblin, and now the quest Necessary Roughness don´t work i can not mount in the robot (Bilgewater Bucaneer), the last release work ok, maybe the change in the core fixed other quest but this dont work it.

**Comment by CrossUCI** [comment]
I download the last snaptshoot  and  the quest Neccesary Roughness with id 24502 is not working , it supose that you have to mount the Bilgewater Bucaneer with id 48526 to begin the event , and other thing the npc Coach Crosscheck with id 37106 is not supose to go up to the ball ?
